GameEngine/GameBoard.cs

```diff
@@ -1,45 +1,41 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Linq;
 
 namespace GameEngine
 {
     public class GameBoard
     { 
         public List<BoardPositionType> Board { get; private set; }
-        public int OwlPosition { get; set; }
-
-
-        public GameBoard(int gameSizeMultiplier)
+        public List<int> OwlPositions { get; set; }
+        
+        public int NestPosition
+        {
+            get { return Board.Count - 1; }
+        }
+        
+        public GameBoard(int gameSizeMultiplier, int numberOfOwls = 6)
         {
             BuildBoard(gameSizeMultiplier);
-            OwlPosition = 0;
+            OwlPositions = Enumerable.Repeat(0, numberOfOwls).ToList();
```

Comment:
❌ The Owls do not all start on the same space. They start in positions 0 to N.

Reply:
I have updated the tests and implementation to reflect this.